src/main/java/com/xceptance/xlt/report/util/MovingArerage.java

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) 2005-2022 Xceptance Software Technologies GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.xceptance.xlt.report.util;
+
+/**
+ * Hold the data for additional moving averages.
+ */
+public class MovingArerage
+{
+    /**
+     * Additional moving averages. 
+     */
+    public enum MovingAverages
+    {
+        /**
+         * Percentage of all values.
+         */
+        PERCENTAGE_OF_VALUES,
+        
+        /**
+         * Amount of time for moving average.
+         */
+        TIME_TO_USE,
+        
+        /**
+         * Amount of requests for moving average.
+         */
+        AMOUNT_OF_REQUESTS;
+    }
+    
+    /**
+     * The value of used for requests and percentage. 
+     */
+    private int intValue = 0;
+    
+    /**
+     * The value used for the time.
+     */
+    private long longValue = 0;
+    
+    /**
+     * Given property value for the time. 
+     */
+    private String timeString = "";
+    
+    /**
+     * Current used type for moving average.
+     */
+    private MovingAverages type;
```

Comment:
In case the values are not expected to change (as the absence of any setter method implies) once they have been set via c'tor, they should be declared as `final`. 
And, there is no need to initialize fields with Java defaults (`int` and `long` are initialized implicitly w/ 0).

BTW do we really need two fields here? IMO having one field of type `long` would suffice too.